skeletons/INTEGER.h

```diff
@@ -43,6 +43,8 @@ xer_type_encoder_f INTEGER_encode_xer;
 per_type_decoder_f INTEGER_decode_uper;
 per_type_encoder_f INTEGER_encode_uper;
 
+#define INTEGER_constraint asn_generic_no_constraint
```

Comment:
Just put a record in case anyone is interested in this discussion. Please refer to [here](https://github.com/mouse07410/asn1c/pull/2#issuecomment-288065551)